app/src/lib/stores/app-store.ts

```diff
@@ -2398,7 +2398,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
         })
       }
 
-      return result || false
+      return !!result
```

Comment:
Like above, explicit comparisons when there's ambiguity in the truthy/falsy checks

```suggestion
      return result !== undefined
```